docs/release-notes/v2.0.rst

```diff
@@ -1,7 +1,242 @@
+.. include:: ../references.txt
+
+
 .. _gammapy_2p0_release:
 
-2.0 (unreleased)
+2.0 (August XX, 2025)
 ----------------
 
 
-- No changes yet
+- Released August XX, 2025
+- 25 contributors
+- 235 (XX)  pull requests since v1.3 (not all listed below)
+- 100 (XX) closed issues
+
+Summary
+~~~~~~~
+
+This is a new Long term stable release. Gammapy 2.0 will be maintained for the next two years, and we removes support for
+Gammapy 1.x versions. With Gammapy 2.0, we provide analysis support for major Imaging Atmospheric Cherenkov Telescope like H.E.S.S.,
```

Comment:
Minor: there are these random line breaks, please remove consistently. Line breaks in md/rst documents are typically used to separate paragraphs. The actual line breaks are then introduced by presentation in the browser.